src/media/local-display-track.ts

```diff
@@ -1,6 +1,6 @@
-import { LocalTrack } from './local-track';
+import { LocalVideoTrack } from './local-video-track';
 
 /**
  * Represents a local track for a display source.
  */
-export class LocalDisplayTrack extends LocalTrack {}
+export class LocalDisplayTrack extends LocalVideoTrack {}
```

Comment:
as you're adding LocalDisplayTrack in this PR, can you also add the LocalComputerAudioTrack too?

Reply:
i thought we agreed on using the LocalAudio Track as it is the same there are no extra property which are different from localAudioTrack

Reply:
I think the issue is in WCME that it needs to detect if a published track is a microphone or screen share audio one - it's doing it using `instanceof` and so we need a separate class, because both mic and screen share audio would return true for `instanceof LocalAudioTrack`

Reply:
correct created a new class for LocalComputer audio now